gst/nnstreamer/include/nnstreamer_cppplugin_api_filter.hh

```diff
@@ -176,7 +176,11 @@ class tensor_filter_subplugin
 
   virtual void invoke (const GstTensorMemory *input, GstTensorMemory *output) = 0;
   /**< Mandatory virtual method.  Invoke NN.
-
+   *
+   * In the case of a dynamic invoke, output tensors information may be changed.
+   * When there's the change, subplugin should update the tensors information in the invoke function.
+   * The tensor filter gets the changed information from the subplugin using getModelInfo().
```

Comment:
Please add a simple guide/example code sniplet how to use it for flexible-tensor-ed invoke. It isn't straightforward as static tensors.

Reply:
I see. Since the filter cpp subplugin is different from the context of this PR, I upload it in the new PR.